compositor_render/src/transformations/web_renderer/render_website.wgsl

```diff
@@ -0,0 +1,68 @@
+struct VertexInput {
+    @location(0) position: vec3<f32>,
+    @location(1) tex_coords: vec2<f32>,
+    @location(2) texture_id: i32,
+}
+
+struct VertexOutput {
+    @builtin(position) position: vec4<f32>,
+    @location(0) tex_coords: vec2<f32>,
+    @location(1) @interpolate(flat) texture_id: i32,
+}
+
+struct CommonShaderParameters {
+    time: f32,
+    texture_count: u32,
+    output_resolution: vec2<u32>,
+}
+
+struct TextureInfo {
+     is_website_texture: i32,
+     transformation_matrix: mat4x4<f32>,
+}
```

Comment:
How does memory alignment work here? Have you tested it?

Reply:
I've had some troubles with getting it right but from what I found out each field in this struct needs 16 byte alignment. `i32` is 4 bytes so I had to add 12 byte padding between `is_website_texture` & `transformation_matrix`